README.md

```diff
@@ -564,12 +569,6 @@ ignored. If this happens, you will need to reapprove the pull request.
 
 This feature has [security implications](#update-merge-conflicts).
 
-#### Private Repositories
-
-`policy-bot` works with private repositories, but currently does not support
-pull requests from private _forks_ of private repositories due to GitHub API
-limitations. Please file an issue if this functionality is important to you.
-
```

Comment:
removing this, because I don't think it's true anymore. private forks of private repos seem to work fine on my end.

Reply:
Interesting, this was originally added due to [this fallback condition](https://github.com/palantir/policy-bot/blob/develop/pull/github.go#L801-L809) when loading commit pushed dates. Maybe GitHub improved the GraphQL API so that pushed dates are returned from forks now and we don't need the fallback.